docs/team/tlylt.md

```diff
@@ -3,66 +3,71 @@ layout: page
 title: Yongliang's Project Portfolio Page
 ---
 
-## Project: imPoster
+<h1 class="post-title">{{ page.title | escape }}</h1>
 
-imPoster, imPoster, Your new API Tester!
+### Project: imPoster
 
-Given below are my contributions to the project.
+imPoster is a desktop application for beginners of API development to easily grasp the basics. The application is optimised for fast typists and can be fully operated through a Command Line Interface. My contributions to the project are as listed below:
 
-\*Below is the template that will be updating overtime, stay tuned!
+**Code contributed**: [RepoSense link](https://nus-cs2103-ay2021s2.github.io/tp-dashboard/?search=tlylt&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-02-19&tabOpen=true&tabType=zoom&zA=tlylt&zR=AY2021S2-CS2103T-T12-4%2Ftp%5Bmaster%5D&zACS=121.05555555555556&zS=2021-02-19&zFS=tlylt&zU=2021-03-29&zMG=undefined&zFTF=commit&zFGS=groupByRepos&zFR=false)
 
-- **New Feature**: Added the ability to undo/redo previous commands.
+#### New Features & Enhancements
 
-  - What it does: allows the user to undo all previous commands one at a time.
-    Preceding undo commands can be reversed by using the redo command.
+- **New Feature**: Added the ability to send a request without saving the endpoint.
+
+  - What it does: allows the user to run API requests on the fly. 
   - Justification: This feature improves the product significantly because a
-    user can make mistakes in commands and the app should provide a convenient
-    way to rectify them.
-  - Highlights: This enhancement affects existing commands and commands to be
-    added in future. It required an in-depth analysis of design alternatives.
+    user can quickly type in an API URL with specifying details, and the app 
+    will conveniently trigger a request for the user to view the response.
+  - Highlights: This enhancement affects existing commands that triggers an outbound
+    request. It required an in-depth analysis of design alternatives.
     The implementation too was challenging as it required changes to existing
-    commands.
-  - Credits: _{mention here if you reused any code/ideas from elsewhere or if a
-    third-party library is heavily used in the feature so that a reader can make
-    a more accurate judgement of how much effort went into the feature}_
+    commands and how we verify the validity of URLs.
+
+- **New Feature**: Added a keyboard shortcut that allows the user to retrieve 
+  the previous valid command using ctrl + up-arrow (Windows) / cmd + up-arrow (Mac).
```

Comment:
Just a suggestion but you may want to consider using the `<kbd>` tag so that the keyboard input are nicely formatted like this: <kbd>ctrl</kbd> + <kbd>up</kbd>

Reply:
Thank you, resolved in my updated commit